### PR TITLE
Update documentation, elucidate the use of filters (and, or).

### PR DESCRIPTION
**Pull request by xdanielsb**
Hi,

Concerning the issue #251 that I closed some minutes ago, I update the docs to clarify the usage of filters to generate filters type (or, and).

Regards,

### CODE REVIEW

File: docs/tutorial.rst [hunk 1]
```diff
@@ -293,13 +293,24 @@ Let's tag some students. ::
...
 
-Now that we have extra meta-data on our rows, we can easily filter our :class:`Dataset`. Let's just see Male students. ::
+Now that we have extra meta-data on our rows, we can easily filter our :class:`Dataset`. Let's just see Female students. ::
 
+    >>> students.filter(['female']).yaml
+    - {first: Bessie, Last: Monke}
 
-    >>> students.filter(['male']).yaml
-    - {first: Kenneth, Last: Reitz}
+By default, when you pass a list of tags you get filter type or. ::
```
Comment (claudep) [inline docs/tutorial.rst]: I think putting backquotes around `or` (and `and` below) would help readability.

**Comment by claudep** [comment]
Thanks for the patch!
If you commit the patch yourself, may I ask you to squash commits before (either manually or by choosing `Squash and commit` button)?
It would be great if you could also reference the issue number in the commit message.

**Comment by xdanielsb** [comment]
sorry @claudep  I made a mistake I did the squash and commit, and push to the wrong branch 'master', I did the rebase in the branch master to try to solve this problem, is there any way to solve that without doing a pull request again?
I dont know if you can do the rebase 
the new commit  name that I did was : "Elucidate the use of filters (and, or), this closes issue #251."

**Comment by claudep** [comment]
The commit is done, so no need to do anything else. Just for the next time, try to squash or just use the "Squash" merge button :-)

**Comment by xdanielsb** [comment]
thanks @claudep, sorry for that nuissance commit

**Comment by hugovk** [comment]
To be on the safe side, shall we protect master from direct commits? 

https://help.github.com/en/github/administering-a-repository/configuring-protected-branches

File: docs/tutorial.rst (hunk 1 above)
Comment (hugovk) [reply docs/tutorial.rst]: Agreed.

**Comment by claudep** [comment]
We could do that, but I don't have access to repo settings.

**Comment by hugovk** [comment]
@jezdez Please could you protect the master branch for this project?

**Comment by jezdez** [comment]
@hugovk Done. I've only enabled the general protection against force-pushes etc. Do you want me to enable other protections?

Screenshot_2019-10-30 jazzband tablib

**Comment by jezdez** [comment]
Oh, and I also disabled the "merge commit" merge option for the repo, so that only rebase and squash commits are allowed:

Screenshot_2019-10-30 jazzband tablib(1)

**Comment by claudep** [comment]
@jezdez Thanks!

**Comment by hugovk** [comment]
Looks good, thanks!

**Pull request merged**